src/Analyzers/MSTest.Analyzers/UseParallelizeAttributeAnalyzer.cs

```diff
@@ -75,6 +80,23 @@ private static void AnalyzeCompilation(CompilationAnalysisContext context)
             }
         }
 
+        if (hasParallelizeAttribute && hasDoNotParallelizeAttribute)
+        {
+            // Both attributes are present - this is an error
+            context.ReportNoLocationDiagnostic(DoNotUseBothAttributesRule);
```

Comment:
Should we produce a diagnostic with location pointing to one of the two declarations of the attribute?